docs/src/test-parameterize-js.md

```diff
@@ -7,7 +7,7 @@ You can either parameterize tests on a test level or on a project level.
 
 <!-- TOC -->
 
-## Parametrized Tests
+## Parameterized Tests
```

Comment:
Aren't both `Parametrized` and `Parameterized` legit?

Reply:
I don't think `Parametrized` is actually a word - I tried googling it

Reply:
Parameterized seems more often used, so landing this patch.